### ISSUE TITLE
Scopy: missing top menu bar

### ISSUE THREAD

**Issue by acusco** [analogdevicesinc/scopy#1376]
**Environment:**
 - OS: Linux (Ubuntu 22.04)
 - FW Version: v0.28
 - SW Version or git tag:  1.4.1-f4beeb1

**Describe the bug**
When running scopy, the top menu bar including the minimize, restore and close buttons are missing. 
Also, dragging or moving the window around not working.

**Screenshots**
![scopy_top_menu_bar](https://user-images.githubusercontent.com/115008598/196686674-36b2b022-312f-44f8-ac92-4a71c7eaf994.png)

**Comment by Modiug** [comment]
Same here.

I wonder if it might be related to the use of Wayland. [Something similar was reported for MPV](https://ubuntuhandbook.org/index.php/2022/06/mpv-no-title-bar-border-ubuntu-2204/).

**Comment by Modiug** [comment]
Indeed, switching Ubuntu to one of the "on Xorg" desktops (and not using Wayland) will bring back the window decoration. However, that's only a workaround.

**Comment by adisuciu** [comment]
I'm not sure if this is our bug or Wayland bug or both. We tried fixing this previously, but it seems it didn't stick ...

Regardless, workaround is running Scopy like this: 

XDG_SESSION_TYPE=x11 flatpak run org.adi.Scopy

We will take another look before a new release.
-Adrian
